### PR TITLE
[sdks] Build the runtimes and LLVM in parallel

### PR DESCRIPTION
**Pull request by luhenry**

### CODE REVIEW

**Comment by luhenry** [comment]
Waiting for https://github.com/mono/mono/pull/10299

File: src/mono-runtimes/mono-runtimes.targets [hunk 1]
```diff
@@ -388,9 +392,12 @@
   <Target Name="_BuildRuntimes"
       DependsOnTargets="_BuildRuntimesWithoutMonoSdks;_GetRuntimesOutputItems"
       Outputs="@(_RuntimeSource);@(_MonoBinarySource);@(_ProfilerSource);@(_MonoPosixHelperSource);@(_RuntimeEglibHeaderSource);@(_MonoBtlsSource);@(_BclTestOutput)">
+    <ItemGroup>
+      <_PackageRuntime Include="package-android-%(_MonoRuntime.Identity)" />
+    </ItemGroup>
     <Exec
         Condition=" '%(_MonoRuntime.UseMonoSdks)' != 'False' And '%(_MonoRuntime.DoBuild)' == 'true' "
-        Command="make DISABLE_IOS=1 $(MakeConcurrency) package-android-%(_MonoRuntime.Identity) $(_MonoSdksParameters)"
+        Command="make DISABLE_IOS=1 $(MakeConcurrency) @(_PackageRuntime, ' ') $(_MonoSdksParameters)"
```
Comment (jonpryor) [inline src/mono-runtimes/mono-runtimes.targets]: This should be:

```
@(_PackageRuntime->'%(Identity)', ' ')
```

File: src/mono-runtimes/mono-runtimes.targets [hunk 2]
```diff
@@ -166,8 +166,12 @@
       DependsOnTargets="_SetAutogenShTimeToLastCommitTimestamp;_PrepareLlvmItems"
       Inputs="$(LlvmSourceFullPath)\Makefile.config.in"
       Outputs="@(_LlvmSourceBinary)">
+    <ItemGroup>
+      <_PackageLlvm Include="package-llvm-%(_LlvmRuntime.Identity)" />
+      <_PackageOptionsLlvm Include="%(_LlvmRuntime.PackageOptions)" />
+    </ItemGroup>
     <Exec
-        Command="make $(MakeConcurrency) package-llvm-%(_LlvmRuntime.Identity) $(_MonoSdksParameters) %(_LlvmRuntime.PackageOptions)"
+        Command="make DISABLE_IOS=1 $(MakeConcurrency) @(_PackageLlvm, ' ') $(_MonoSdksParameters) @(_PackageOptionsLlvm, ' ')"
```
Comment (jonpryor) [inline src/mono-runtimes/mono-runtimes.targets]: This should be:

```
@(_PackageRuntime->'%(Identity)', ' ')
```

File: src/mono-runtimes/mono-runtimes.targets (hunk 1 above)
Comment (luhenry) [reply src/mono-runtimes/mono-runtimes.targets]: Ok

File: src/mono-runtimes/mono-runtimes.targets (hunk 2 above)
Comment (luhenry) [reply src/mono-runtimes/mono-runtimes.targets]: Ok

**Comment by luhenry** [comment]
@jonpryor fixed.

**Comment by luhenry** [comment]
This shaves off around 25minutes from the Ubuntu PR build:
 - https://jenkins.mono-project.com/job/xamarin-anroid-linux-pr-builder/3494/ -> 1 hour <-- *without* the change
 - https://jenkins.mono-project.com/job/xamarin-anroid-linux-pr-builder/3482/ -> 59 min <-- *without* the change
 - https://jenkins.mono-project.com/job/xamarin-anroid-linux-pr-builder/3499/ -> 36 min <-- *with* the change

**Pull request merged**